src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilterTest.java

```diff
@@ -289,6 +289,37 @@ public void testInvalidMessageFormat() throws Exception {
         }
     }
 
+    @Test
+    public void testInvalidMessageFormatWithoutOff() throws Exception {
```

Comment:
this test covers an edge case where the `messageFormat` might be invalid but is not validated if there is only an "on" comment